src/scales/quantitative.js

```diff
@@ -206,7 +207,7 @@ export function ScaleQuantize(
     if (min instanceof Date) thresholds = thresholds.map((x) => new Date(x)); // preserve date types
   }
   if (order(arrayify(domain)) < 0) thresholds.reverse(); // preserve descending domain
-  return ScaleThreshold(key, channels, {domain: thresholds, range, reverse});
+  return ScaleThreshold(key, channels, {domain: thresholds, range, reverse, unknown});
```

Comment:
This fixes an (unfiled) bug where the quantize scale didn’t respect the **unknown** option.